internal/pkg/ecs/ecs.go

```diff
@@ -232,6 +246,37 @@ func (c Client) StopDefaultClusterTasks(familyName string) error {
 	return c.ecsClient.StopTasks(taskIDs, ecs.WithStopTaskReason(taskStopReason))
 }
 
+// DecryptedSecrets returns the decrypted parameters from either the SSM store or Secrets Manager.
+func (c Client) DecryptedSecrets(secrets []*ecs.ContainerSecret) ([]EnvVar, error) {
+	var ssmSecrets []EnvVar
+	var secretManagerSecrets []EnvVar
+	for _, secret := range secrets {
+		parsed, err := arn.Parse(secret.ValueFrom)
+		if err != nil || parsed.Service == ssm.Namespace {
```

Comment:
why are we ignoring the error from parsing the arn?

Reply:
If in the Task Definition a secret doesn't have arn as its value and have something like `/copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/my-secret` as the value we get an error. So in that case we are assuming it to be a value from ssm parameter store.

Reply:
i think secrets in a task definition _have_ to be ARNs, right? https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_Secret.html

> **valueFrom**
The secret to expose to the container. The supported values are either the full ARN of the AWS Secrets Manager secret or the full ARN of the parameter in the SSM Parameter Store.